spark-connector/spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/catalog/BaseCatalog.java

```diff
@@ -65,9 +65,9 @@ public abstract class BaseCatalog implements TableCatalog, SupportsNamespaces {
   protected Catalog gravitinoCatalogClient;
   protected PropertiesConverter propertiesConverter;
   protected SparkTransformConverter sparkTransformConverter;
+  protected final String metalakeName;
```

Comment:
no need to protected if moving `loadGravitinoTable`?

Reply:
yes. have rolled it back.